### PR TITLE
Cyble enhancements

### PR DESCRIPTION
**Pull request by cyble-dev**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- We have made changes in our IOCs response, this release of Cortex app implements these changes.
- A minor bug fix in Alerts fetching code.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by cyble-dev** [comment]
Hello @israelpoli, @edik24,
Please proceed with reviewing the PR.
This is a critical release and we need it live as soon as possible.

Thanks.

File: Packs/CybleEventsV2/ReleaseNotes/1_0_4.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#### Integrations
+
+##### CybleEvents v2
+
```
Comment (israelpoli) [inline Packs/CybleEventsV2/ReleaseNotes/1_0_4.md:4]: Please add a message regarding the new limit (100 instead of 1000)

**Review by israelpoli — COMMENTED** [review]
Hi @cyble-dev looks great,
See below one comment

**Comment by cyble-dev** [comment]
Hello @israelpoli,
I have made the changes. Please proceed.

**Pull request merged**